docs/docs/03-configuration.md

```diff
@@ -87,6 +87,7 @@ Either `OPENAI_API_KEY` or `OLLAMA_BASE_URL` need to be set for automatic taggin
 | CRAWLER_VIDEO_DOWNLOAD_MAX_SIZE    | No       | 50      | The maximum file size for the downloaded video. The quality will be chosen accordingly. Use -1 to disable the limit.                                                                                                                                                                                                                                                          |
 | CRAWLER_VIDEO_DOWNLOAD_TIMEOUT_SEC | No       | 600     | How long to wait for the video download to finish                                                                                                                                                                                                                                                                                                                             |
 | CRAWLER_ENABLE_ADBLOCKER           | No       | true    | Whether to enable an adblocker in the crawler or not. If you're facing troubles downloading the adblocking lists on worker startup, you can disable this.                                                                                                                                                                                                                     |
+| YTDLP_ARGS           | No       | []    | Include additional YTDLP arguments to be passed at crawl time: https://github.com/yt-dlp/yt-dlp?tab=readme-ov-file#general-options
```

Comment:
Can you wrap the link in a markdown link? Also mention that args are %% separated? The default is also empty string not `[]` as it might be confusing to some people.
```suggestion
| CRAWLER_YTDLP_ARGS                 | No       | Not set | Include additional YTDLP arguments to be passed at crawl time. Different args should be separated by "%%". Reference for all the args is [here](https://github.com/yt-dlp/yt-dlp?tab=readme-ov-file#general-options)                                                                                                                                                          |
```